src/core/namespace.c

```diff
@@ -64,6 +64,20 @@ typedef struct BindMount {
         bool ignore;
 } BindMount;
 
+typedef struct TargetMount {
+        const char *path;
+        MountMode mode;
+        bool ignore;
+} TargetMount;
+
+/* ProtectKernelTunables= option and the related filesystem APIs */
+static const TargetMount protect_kernel_tunables_table[] = {
+        { "/proc/sys",                  READONLY,       false, },
+        { "/proc/sysrq-trigger",        READONLY,       true, /* Not always compiled into kernel */ },
+        { "/sys",                       READONLY,       false, },
+        { "/sys/fs/cgroup",             READWRITE,      false, }, /* READONLY is set by ProtectControlGroups= option */
```

Comment:
Maybe it's just OCD, but the way that one comment is outside, while the other is it's own separate argument after a comma pains me :) Maybe you could make the whitespace a bit more frugal and remove the trailing commas?